src/circuit/gadget/poseidon/pow5t3.rs

```diff
@@ -177,10 +187,19 @@ impl<F: FieldExt> Pow5T3Chip<F> {
             };
 
             vec![
-                pad_and_add(initial_state_0, input_0, output_state_0),
-                pad_and_add(initial_state_1, input_1, output_state_1),
+                (
+                    "state[0]",
```

Comment:
```suggestion
                    "state[0] pad_and_add",
```

Reply:
The gate is named "padd-and-add", so this is duplicative and unhelpful to debugging.

Reply:
It depends whether dev tools always show the gate name, I guess.